File: src/Test/java/com/chengte99/StudentTest.java
package com.chengte99;

import com.chengte99.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void highestTest() {
        Student student = new Student("Tom", 69, 80);
        Assertions.assertEquals(80, student.highest());
    }

    @Test
    public void averageTest() {
        Student student = new Student("Tom", 69, 80);
        Assertions.assertEquals((69+80)/2, student.getAverage());
    }
}

File: src/Test/kotlin/com/chengte99/StudentTest.kt
package com.chengte99.kotlin

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    @Test
    fun grading() {
        val s = Student("Kevin", 60, 80)
        Assertions.assertEquals('C', s.grading())
    }
}